tests/installation/iso_checksum.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify data integrity of the iso image by comparing checksum from IBS/OBS with checksum on the worker
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use Digest::file 'digest_file_hex';
+
+sub run {
+    my $self = shift;
+
+    my $result       = 'fail';
+    my $iso          = get_var("ISO");
+    my $checksum     = get_var("ISO_CHECKSUM");
```

Comment:
as above but you can skip the scheduling of the whole module if this variable is not set